File: Java/src/main/java/com/leanrose/Battery.java
package com.leanrose;

public class Battery {

    public String type;

    public boolean dry;

    public double charge;

    public Battery(String type, boolean dry, double charge) {
        this.type = type;
        this.dry = dry;
        this.charge = charge;
    }

    @Override
    public String toString() {
        return this.type + ", " + this.dry + ", " + this.charge;
    }
}

File: CSharp/ElectricRoseBatteryTracker/Models/Battery.cs
namespace ElectrictRoseBatteryTracker.Models;

public class Battery
{
    public string Type { get; init; }
    public bool Dry { get; set; }
    public double Charge { get; set; }

    public Battery(string type, bool dry, double charge)
    {
        Type = type;
        Dry = dry;
        Charge = charge;
    }
}